source/manual/taxonomy.html.md

```diff
@@ -54,15 +54,22 @@ content item for this guidance document][example-guidance].
 ## Accessing the taxonomy
 
 The level one taxons are associated with the GOV.UK home page through
-the `level_one_taxons` link type.
+the `root_taxon` link type. The GOV.UK home page in turn has a 
+corresponding reverse link of link type `level one taxons`.
 
-This is the content item for the top-level "Education" taxon:
+This is the content item for the GOV.UK home page with all level one
+taxons listed.
+
+[https://www.gov.uk/api/content/] [govuk-taxon]
```

Comment:
The space between the square brackets is preventing this from being rendered as a link, but also the `govuk-taxon` doesn't have a reference like other links in the bottom of this page

Reply:
fixed